app/controllers/health_checks_controller.rb

```diff
@@ -0,0 +1,25 @@
+class HealthChecksController < ApplicationController
+  skip_before_action :authenticate_user!, only: :index
+  before_action :skip_authorization, only: :index
+
+  def index
+    [
+      Thread.new { check_postgres },
+      Thread.new { check_redis },
+    ].each(&:join)
+
+    head :ok
+  end
+
+  private
+
+  def check_postgres
+    ActiveRecord::Base.logger.silence do
```

Comment:
This is particularly so that we don't see ActiveRecord query logs in development.